tools/schemacode/src/bidsschematools/__main__.py

```diff
@@ -149,9 +149,6 @@ def pre_receive_hook(schema, input_, output):
         )
 
     regexes = [rule["regex"] for rule in all_rules]
-    # XXX Hack for phenotype files - this can be removed once we
-    # have a schema definition for them
-    regexes.append(r"phenotype/.*\.(tsv|json)")
```

Comment:
and we do not need any extra rules because there is already
https://github.com/bids-standard/bids-specification/blob/HEAD/src/schema/rules/files/common/tables.yaml#L34
?

Reply:
Correct